helpers/test/dismissible.test.js

```diff
@@ -66,7 +66,7 @@ describe('dismissible', () => {
 	});
 
 	[undefined, null, 0, 'hello', []].forEach((cb) => {
-		it(`should handle invalid callback: ${cb}`, () => {
+		it(`should handle invalid callback: ${JSON.stringify(cb)}`, () => {
```

Comment:
Currently test reporting doesn't allow leading or trailing spaces so we can have the name be consistent. We've had a few come up were the name was the same minus spaces. There are future changes that will probably make this a non-issue but for now changing to `stringfy` the `cb` for the test name since `[]` default serialize to just nothing meaning the test name is `should handle invalid callback: ` with a trailing space.